manim/mobject/number_line.py

```diff
@@ -68,6 +68,49 @@ class NumberLine(Line):
     kwargs : Any
         Additional arguments to be passed to :class:`~.Line`.
 
+    Examples
+    --------
+    .. manim:: NumberLineExample
+        :save_last_frame:
+
+        class NumberLineExample(Scene):
+            def construct(self):
+                l0 = NumberLine(
+                    x_range=[-10, 10, 2],
+                    length=10,
+                    color=BLUE,
+                    include_numbers=True,
+                    label_direction=UP,
+                )
+
+                l1 = NumberLine(
+                    x_range=[-10, 10, 2],
+                    unit_size=0.5,
+                    numbers_with_elongated_ticks=[-2, 4],
+                    include_numbers=True,
+                    number_scale_value=0.5,
+                )
+                num6 = l1.get_number_mobject(6).set_color(RED)
```

Comment:
`get_number_mobject` creates a new label with 6, as far as I know -- would something slightly more complicated like

```suggestion
                [num6] = [num for num in l1.numbers if num.number == 6]
                num6.set_color(RED)
```

make sense? Then the existing label for 6 is found and modified, and `num6` wouldn't have to be added again.